.github/workflows/new_issue_comment.yml

```diff
@@ -0,0 +1,20 @@
+name: Process new comment on issue
+
+on: 
+  pull_request: # TODO change with issue_comment
+
+jobs:
+  wti:
+    name: Run wti
+    runs-on: windows-2022
+    permissions:

```

Comment:
The logic is that WTI only comment with all the debugging information that it capture (under 'diagnostics information') and if it doesn't find a definite answer, then it appends the 'similar issues' block. 

In the case where WTI doesn't find anything at all, I'd still like to see an empty message just to show that nothing has been found so that I know that WTI is not running / pending

Reply:
Also added logic in WTI to skip the similar issue block if empty: https://github.com/OneBlue/wti/commit/ec216e12d337f6422e75cb7908169ba99c0ad90e

Reply:
Hmm I can't quite follow the logic here, but as long as you're accounting for the case where both the log is blank and the similar issues is blank then we're good! :)